src/pages/organize/[orgId]/projects/[campId]/areaassignments/[areaAssId]/report.tsx

```diff
@@ -381,6 +400,71 @@ const AreaAssignmentReportPage: PageWithLayout<AreaAssignmentReportProps> = ({
             </Box>
           </Box>
           <Box ml={2} width="40%">
+            {assignment.start_date && (
+              <ZUICard
+                header={
+                  !unlocked ? (
+                    <Msg id={messagesIds.report.lockCard.header} />
+                  ) : (
+                    <Msg id={messagesIds.report.lockCard.headerUnlock} />
+                  )
+                }
+                status={
+                  <Switch
+                    checked={!unlocked}
+                    onChange={(event) => setUnlocked(!event.target.checked)}
+                  />
+                }
+                subheader={
+                  !unlocked
+                    ? messages.report.lockCard.description()
+                    : messages.report.lockCard.descriptionUnlock()
+                }
+                sx={{ mb: 2 }}
+              >
+                {unlocked && (
+                  <Box>
+                    <Divider />
+                    <Typography my={1}>
+                      <Msg id={messagesIds.report.lockCard.safe} />
+                    </Typography>
+                    <Box alignItems="center" display="flex">
+                      <Check style={{ color: theme.palette.success.main }} />
+                      <Typography ml={1}>
+                        <Msg id={messagesIds.report.lockCard.fix} />
+                      </Typography>
+                    </Box>
+                    <Box alignItems="center" display="flex">
+                      <Check style={{ color: theme.palette.success.main }} />
+                      <Typography ml={1}>
+                        <Msg id={messagesIds.report.lockCard.reorder} />
+                      </Typography>
+                    </Box>
```

Comment:
It's currently not possible to reorder questions, so this should probably not be in the list.